aspnetcore/tutorials/first-mvc-app/working-with-sql.md

```diff
@@ -90,35 +97,37 @@ Replace the contents of *Program.cs* with the following code:
 
 [!code-csharp[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie3/Program.cs)]
 
-Test the app
+Test the app.
 
 # [Visual Studio](#tab/visual-studio)
 
-* Delete all the records in the DB. You can do this with the delete links in the browser or from SSOX.
-* Force the app to initialize (call the methods in the `Startup` class) so the seed method runs. To force initialization, IIS Express must be stopped and restarted. You can do this with any of the following approaches:
+Delete all the records in the database. You can do this with the delete links in the browser or from SSOX.
 
-  * Right click the IIS Express system tray icon in the notification area and tap **Exit** or **Stop Site**
+Force the app to initialize, calling the methods in the `Startup` class, so the seed method runs. To force initialization, IIS Express must be stopped and restarted. You can do this with any of the following approaches:
 
-    ![IIS Express system tray icon](working-with-sql/_static/iisExIcon.png)
+* Right-click the IIS Express system tray icon in the notification area and tap **Exit** or **Stop Site**:
 
-    ![Contextual menu](working-with-sql/_static/stopIIS.png)
+  ![IIS Express system tray icon](working-with-sql/_static/iisExIcon.png)
 
-    * If you were running VS in non-debug mode, press F5 to run in debug mode
-    * If you were running VS in debug mode, stop the debugger and press F5
+  ![Contextual menu](working-with-sql/_static/stopIIS.png)
+
+* If running VS in non-debug mode, press F5 to run in debug mode.
+* If running VS in debug mode, stop the debugger and press F5.
```

Comment:
Original better, a rare case I prefer you. Also restore leading white space so you don't get the extra `new line